ARGO/src/ProjectileManager.cpp

```diff
@@ -68,20 +78,36 @@ void ProjectileManager::createEnemyBullet(const createBulletEvent& t_event)
 	}
 }
 
-void ProjectileManager::update(BaseSystem* t_system)
+void ProjectileManager::update(float t_dt)
 {
 	for (auto& bullet : m_playerBullets)
 	{
 		if (static_cast<HealthComponent*>(bullet.entity.getAllComps().at(COMPONENT_ID::HEALTH_ID))->getHealth() > 0)
```

Comment:
This logic and the one for the For loop below can be made into a function where you pass in the specific bullet